chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -91,7 +91,8 @@ def self.bridge?(nic)
   def self.ovs_bridge?(nic)
     nic.kind_of?(::Nic::OvsBridge) or
       # If no ovs tools are installed there's no ovs switch
-      if ::Kernel.system("which ovs-vsctl > /dev/null 2>&1")
+      if ::Kernel.system("which ovs-vsctl > /dev/null 2>&1 && " \
+                         "systemctl -q is-active openvswitch.service")
```

Comment:
I already thought about that and decided that since this is the `master` branch, it is safe enough to assume systemd is running, and better to avoid making assumptions about implementation details (which something like `ps -ef | grep -q ovsdb-server` would do).